load/LoadOrganizations.py

```diff
@@ -96,15 +96,72 @@ def hasDamIds(self, languageRecord, typeCode):
 #			if not name in copyrightSet:
 #				resultSet.add(name)
 
+	def process_other_agreement(self, language_record, type_code, fileset_id):
+		licensors = []
+		if language_record.Has_Ambiguous_Methodology():
+			print("ERROR Agreement Type is 'Other', ambiguous methodology and it needs to be fixed for fileset id: %s" % (fileset_id))
+			return None
+
+		if language_record.HasTraditionalRecording() and type_code == "text":
+			if language_record.Licensor() != None:
+				licensors.append(language_record.Licensor())
+			else:
+				print(f"ERROR Agreement Type is 'Other' and Methodology 'TraditionalRecording' but licensor is empty for fileset id: {fileset_id}")
+			return licensors
+
+		if language_record.HasVirtualRecording() and type_code == "text":
+			if language_record.Licensor() != None:
+				licensors.append(language_record.Licensor())
+			else:
+				print(f"ERROR Agreement Type is 'Other' and Methodology 'VirtualRecording' but licensor is empty for fileset id: {fileset_id}")
+			return licensors
+
+		if language_record.HasPartner():
+			if type_code == "audio":
+				if language_record.Licensor() != None:
+					licensors.append(language_record.Licensor())
+				else:
+					print(f"ERROR Agreement Type is 'Other' and Methodology 'Partner' but licensor is empty for fileset id: {fileset_id}")
+			if type_code == "text":
+				if language_record.CoLicensor() != None:
+					licensors.append(language_record.CoLicensor())
+				else:
+					print(f"WARN Agreement Type is 'Other' and Methodology 'Partner' but co-licensor is empty for fileset id: {fileset_id}")
+			return licensors
+
+		if language_record.HasJoint() and type_code == "text":
+			if language_record.Licensor() != None:
+				licensors.append(language_record.Licensor())
+			else:
+				print(f"ERROR Agreement Type is 'Other' and Methodology 'Joint' but licensor is empty for fileset id: {fileset_id}")
+			return licensors
+
+		if language_record.HearThis():
+			print(f"WARN Agreement Type is 'Other' and Methodology 'Hear This'. We are not able to process for fileset id: {fileset_id}")
+			return None
+
+		if language_record.HasRender() and language_record.Licensor() != None and type_code == "audio":
+			licensors.append(language_record.Licensor())
+			return licensors
+
+		print("WARN Agreement Type is 'Other', which is not currently being processed. licensor: %s and co licensor: %s, fileset id: %s, type code: %s" % (language_record.Licensor(), language_record.CoLicensor(), fileset_id, type_code))
```

Comment:
change this to an ERROR log statement: "AgreementType is Other. Unable to recognize how to process. licensor: xx, co-licensor:xx, fileset: xx, type code: xx"

Reply:
@bradflood , I have changed this message as you can see in the following url: https://github.com/faithcomesbyhearing/dbp-etl/pull/135/files#diff-0628aedffa821d1b9acf4c81d50c756fe72674fcc7ca34c47e0a5b0a65c60aceR162